### PR TITLE
Add physical_disk collector

### PR DESCRIPTION
**Pull request by cbwest3-ntnx**
This PR implements the physical_disk collector, based on the logical_disk collector. This is a result from the request in https://github.com/prometheus-community/windows_exporter/issues/795#issuecomment-860029644.

### CODE REVIEW

File: exporter.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ type prometheusVersion struct {
 }
 
 const (
-	defaultCollectors            = "cpu,cs,logical_disk,net,os,service,system,textfile"
+	defaultCollectors            = "cpu,cs,logical_disk,physical_disk,net,os,service,system,textfile"
```
Comment (breed808) [inline exporter.go:52]: Does it make sense to have both the `logical_disk` and `physical_disk` collectors enabled by default? That is, do the metrics exposed by both collectors overlap in scope?

File: docs/collector.physical_disk.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+`read_bytes_total` | Rate at which bytes are transferred from the disk during read operations | counter | `disk`
+`reads_total` | Rate of read operations on the disk | counter | `disk`
+`write_bytes_total` | Rate at which bytes are transferred to the disk during write operations  | counter | `disk`
+`writes_total` | Rate of write operations on the disk  | counter | `disk`
+`read_seconds_total` | Seconds the disk was busy servicing read requests | counter | `disk`
+`write_seconds_total` | Seconds the disk was busy servicing write requests | counter | `disk`
+`free_bytes` | Unused space of the disk in bytes (not real time, updates every 10-15 min) | gauge | `disk`
+`size_bytes` | Total size of the disk in bytes (not real time, updates every 10-15 min) | gauge | `disk`
+`idle_seconds_total` | Seconds the disk was idle (not servicing read/write requests) | counter | `disk`
+`split_ios_total` | Number of I/Os to the disk split into multiple I/Os | counter | `disk`
```
Comment (breed808) [inline docs/collector.physical_disk.md:36]: Can you add the prefix to the documented metrics? E.G. `windows_physical_disk_requests_queued`

File: collector/physical_disk.go [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+	"regexp"
+	"strings"
+
+	"github.com/prometheus-community/windows_exporter/log"
+	"github.com/prometheus/client_golang/prometheus"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+func init() {
+	registerCollector("physical_disk", NewPhysicalDiskCollector, "PhysicalDisk")
```
Comment (breed808) [inline collector/physical_disk.go:17]: The `init()` function will need to be removed, with collector registration details moved to [init.go](https://github.com/prometheus-community/windows_exporter/blob/master/collector/init.go). I'm happy to make this change if you like.

File: collector/physical_disk.go [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+}
+
+var (
+	diskWhitelist = kingpin.Flag(
+		"collector.physical_disk.disk-whitelist",
+		"Regexp of disks to whitelist. Disk name must both match whitelist and not match blacklist to be included.",
+	).Default(".+").String()
+	diskBlacklist = kingpin.Flag(
+		"collector.physical_disk.disk-blacklist",
+		"Regexp of disks to blacklist. Disk name must both match whitelist and not match blacklist to be included.",
```
Comment (breed808) [inline collector/physical_disk.go:27]: Can we change the flag names from `whitelist` and `blacklist` to `include` and `exclude`, respectively?
See #679 for context.

**Review by breed808 — CHANGES_REQUESTED** [review]
Thanks for submitting this! Is this intended to replace #803?

I've left feedback for a few items. Let me know if any clarification is required.

File: collector/physical_disk.go (hunk 3 above)
Comment (rob-scheepens) [reply collector/physical_disk.go:17]: Hi Ben, yes please make this change for me if you could.

File: collector/physical_disk.go (hunk 4 above)
Comment (rob-scheepens) [reply collector/physical_disk.go:27]: Done in 3c3e40e392c64358e355e2138afc07edae69f511, under PR #803.

File: docs/collector.physical_disk.md (hunk 2 above)
Comment (rob-scheepens) [reply docs/collector.physical_disk.md:36]: Done in 153656f41415eee5166cbbb182d33733a7f141b7 for #803.

Note: logical_disk collector is also missing the prefix.

**Comment by cbwest3-ntnx** [comment]
Closing in favor of #803.

**Pull request closed without merging**